performance_manager/lib/rt_trip_updates.py

```diff
@@ -155,18 +160,115 @@ def get_and_unwrap_tu_dataframe(
     events["is_moving"] = False
     events["pk_id"] = None
 
+    return events
+
+
+def join_gtfs_static(
+    trip_updates: pandas.DataFrame, db_manager: DatabaseManager
+) -> pandas.DataFrame:
+    """
+    join trip update dataframe to gtfs static records
+
+    adds "fk_static_timestamp" and "parent_station" columns to dataframe
+    """
+    # get unique "start_date" values from trip update dataframe
+    # with associated minimum "timestamp_start"
+    date_groups = trip_updates.groupby(by="start_date")["timestamp_start"].min()
+
+    # pylint: disable=duplicate-code
+    # TODO(ryan): the following code is duplicated in rt_vehicle_positions.py
+
```

Comment:
might be able to combine this into a library file, business logic between vehicle positions and trip updates is close, but not exact.